addons/binding/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -307,7 +307,9 @@ end
 
 ### Synchronise two instances
 
-Define a broker and a trigger channel on that broker in a "demo.Things" file:
+To synchronize item items from a SOURCE openHAB instance to a DESTINATION instance, do the following:
```

Comment:
Thanks for the update!

Can you update this readme so only American English is used? AFAIK that's the most commonly used variant of English in computing. The text now contains both American and British variants.

E.g. by replacing the variants of:

synchronise -> synchronize
recognise -> recognize

Reply:
done